### DISCUSSION TITLE
I want to create multiple public APIs.

### DISCUSSION THREAD

**Discussion by akaikite** [oobabooga/text-generation-webui#3381] [Q&A] [answered]
I wanted to provide multiple models to people at the same time, so I ran multiple cmd windows.
However, when I applied the option -api -- public api to multiple webui, it was rejected.
Can anyone tell me how to create multiple APIs for beginners?

OSError: [Errno 10048] error while attempting to bind on address ('127.0.0.1', 5005)

**Comment by missionfloyd** [comment, marked as answer]
You have to use different ports for each one.

**Comment by akaikite** [comment]
I solved the problem by using --api-blocking-port xxxx. Thanks to all.
